src/main/java/com/answerdigital/answerking/repository/ProductRepository.java

```diff
@@ -13,4 +13,6 @@ public interface ProductRepository extends PagingAndSortingRepository<Product, L
     boolean existsByNameAndIdIsNot(String name, Long id);
 
     List<Product> findAll();
+
+    List<Product> findProductsByCategoriesId(Long categoryId);
```

Comment:
same as above, `findProductsByCategoryId` as it needs single category id.

Reply:
I've done the changes.